interfaces/cython/cantera/reaction.pyx

```diff
@@ -353,7 +381,7 @@ cdef class Arrhenius:
             return self.rate.activationEnergy_R() * gas_constant
 
     def __repr__(self):
-        return 'Arrhenius(A={:g}, b={:g}, E={:g})'.format(
+        return '<Arrhenius: A={:g}, b={:g}, E={:g}>'.format(
```

Comment:
What's the motivation for this change? Per the [Python documentation](https://docs.python.org/3.9/library/functions.html?highlight=repr#repr), the angle bracket notation is normally used when the returned string doesn't provide the full information needed to reconstruct the object.

Reply:
This was mostly for aesthetics ... reactions use angle brackets. This specific object will disappear and new objects will use angle brackets (see other comment).

Reverting as it will be moot eventually.